systemtest/src/main/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -1029,8 +1030,76 @@ void checkContainerConfiguration(String podNamePrefix, String containerName, Str
             pods.forEach(pod -> {
                 pod.getSpec().getContainers().stream().filter(c -> c.getName().equals(containerName))
                         .forEach(container -> {
+                            List list = container.getEnv().stream().map((Function<EnvVar, Object>) EnvVar::getName).collect(Collectors.toList());
+                            assertTrue(list.contains(configKey));
+
                             container.getEnv().stream().filter(envVar -> envVar.getName().equals(configKey))
-                                    .forEach(envVar -> assertEquals(config, envVar.getValue()));
+                                    .forEach(envVar -> {
+                                        LOGGER.info("Check specific configuration: {}", envVar.getName());
+                                        for (Map.Entry<String, Object> entry : config.entrySet()) {
+                                            assertThat(envVar.getValue().contains(entry.getKey() + "=" + entry.getValue() + "\n"), is(true));
```

Comment:
So, is this method used only to check that env var with the user suplied properties for the Kafka / Zoo properties file? You should make it more clear. Because up to until this line I expected that container configuration means the fileds of the container and not the KAfka/Zoo configuration.

Reply:
it checks env variable from containers, which is passed into configuration files for bridge/kafka/zoo/mm. So maybe rename it to `checkComponentConfiguration()` and second method to `checkSpecificVariablesInContainer()` ?

Reply:
Yeah ... I think renaming it would help.